.github/workflows/lint.reusable.yml

```
@@ -32,7 +33,7 @@ jobs:
         run: .github/.goassets/scripts/lint-commit-message.sh
 
       - name: File names
-        run: .github/.goassets/scripts/lint-filename.sh
+        run: .github/.goassets/scripts/lint-filename.go
```

I assume we need to use a `go run .github/.goassets/scripts/lint-filename.go` here.
```suggestion
        run: go run .github/.goassets/scripts/lint-filename.go
```